src/wallet/db.cpp

```diff
@@ -402,11 +403,41 @@ CDB::CDB(CWalletDBWrapper& dbw, const char* pszMode, bool fFlushOnCloseIn) : pdb
                             0);
 
             if (ret != 0) {
+                error = strprintf("CDB: Error %d, can't open database %s", ret, strFilename);
```

Comment:
Use same format as below: `CDB: Can't open database %s (open error %d)"`.

Reply:
Didn't do this to avoid changing the existing error message.